modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/handle/migrate/MigrateExportDataProcess.java

```diff
@@ -26,7 +26,7 @@ public class MigrateExportDataProcess implements Serializable, JavaDelegate {
 
     @Autowired
     public void setRuntimeService(RuntimeService runtimeService) {
-        this.runtimeService = runtimeService;
+        MigrateExportDataProcess.runtimeService = runtimeService;
```

Comment:
please make these fields as non-static and autowire via constructor

Reply:
The workflow engine couldn't instantiate class after I change the static class to autowired bean.
![image](https://github.com/eclipse-xpanse/xpanse/assets/121531362/5438382e-3a03-4c8f-98e9-ed1aa1220867)